server/data/interfaces/caseNotesApi/CaseNoteType.ts

```diff
@@ -9,12 +9,11 @@ export default interface CaseNoteType {
 
 export interface CaseNotesTypeParams extends QueryParams {
   dpsUserSelectableOnly?: boolean
-  includeSensitive?: boolean
-  includeRestrictedUse?: boolean
+  includeRestricted?: boolean
   includeInactive?: boolean
 }
 
 export interface CaseNotesTypeQueryParams extends QueryParams {
   selectableBy: 'ALL' | 'DPS_USER'
-  include: ('INACTIVE' | 'SENSITIVE' | 'RESTRICTED')[]
+  include: ('INACTIVE' | 'RESTRICTED')[]
```

Comment:
The include query parameter has been removed from the API

Reply:
fixed